index.html

```diff
@@ -325,7 +325,7 @@ <h6 data-lang="zh-hant">發表對本文檔的建議</h6>
       <p data-lang="en">Chinese characters include Traditional Chinese and Simplified Chinese alternatives. The former is commonly used in Taiwan, Hong Kong and Macao while the latter is commonly used in Mainland China, Singapore and Malaysia.</p>
       <p data-lang="zh-hans">依照各地的通行标准，中文所使用的汉字主要分为繁体字与简体字。前者又称正体字、传统汉字等，主要通行于台湾、香港、澳门等地；后者又作简化字，主要通行于中国大陆、星马地区。本文依笔画多寡与部件结构为区别，统称为繁体字与简体字。</p>
       <p data-lang="zh-hant">依照各地的通行標準，中文所使用的漢字主要分為繁體字與簡體字。前者又作正體字、傳統漢字等，主要通行於台灣、香港、澳門等地；後者又作簡化字，主要通行於中國大陸、星馬地區。本文依筆畫多寡與部件結構為區別，統稱為繁體字與簡體字。</p>
-      <p data-lang="en">Different glyphs are used in different regions. One Unicode code point of a Chinese character may have more than one valid glyph, depending on operating systems and typefaces. The focus of this document is Chinese composition and will not discuss more about glyph variations.</p>
+      <p data-lang="en">Different glyphs are used in different regions. One Unicode code point of a Chinese character may have more than one valid glyph, depending on operating system and typeface used. The focus of this document is Chinese composition and will not discuss cover glyph variations in detail.</p>
```

Comment:
> The focus of this document is Chinese composition and will not discuss cover glyph variations in detail

maybe say

The focus of this document is Chinese composition and it will not cover glyph variations in detail